### ISSUE TITLE
NER all result is 'O' label, why ? ask help

### ISSUE THREAD

**Issue by ysgncss** [ThilinaRajapakse/simpletransformers#452]
this a output example:
O 0.9995051 [0.9995051, 7.186927e-05, 8.351679e-05, 5.8512996e-05, 4.1969375e-05, 5.33289e-05, 6.2319625e-05, 5.4373875e-05, 6.933002e-05]

**Comment by ThilinaRajapakse** [comment]
Hard to say without more information. But some general advice would be to try lowering the learning rate and the number of training epochs.

**Comment by plc-dev** [comment]
I've had a similar issue as well some time ago and opted for a different framework back then, but I decided to give it another shot today.
The issue for me seemed to be that my dataset was split with a tabular "/t" instead of a singular whitespace as called for in the [conll example](https://towardsdatascience.com/simple-transformers-named-entity-recognition-with-transformer-models-c04b9242a2a0) .
It seems odd that the torch dataloader doesn't accept both delimiters but that is the only thing I recall changing.
